src/containers/header/burgerMenu/index.js

```diff
@@ -1,27 +1,26 @@
 import React, { Component } from 'react';
 import { MdSearch } from 'react-icons/md';
-// conect after setting router and change links,
-// import { NavLink } from 'react-router-dom';
 import styles from './styles.module.scss';
```

Comment:
Maybe it is better to use TODO comments. It looks better to get attention on this comment TODO to make some deals.

for example 

/**
* TODO: needs to do something
*/